cdk/config.ts

```diff
@@ -67,6 +68,9 @@ const createConfig = (stage: appEnv): Config => ({
   domainName: domainNames[stage],
   zoneName,
   certificateARN: <string>process.env.CERTIFICATE_ARN,
+  // Add this ARN if you need to attach a basic authentication to your app
+  // before the actual release date
+  basicAuthLambdaARNWithVersion: process.env.BASIC_AUTH_LAMBDA_ARN,
```

Comment:
How are we disabling the Basic Auth, do we need the deployment for the same?

Reply:
We are not open for public users, so we are restricting the mainnet access @ksridharbabuus